src/foundation.cs

```diff
@@ -3173,6 +3173,12 @@ public interface NSExpression : NSSecureCoding, NSCopying {
 		[Static, Export ("expressionForFunction:arguments:")]
 		NSExpression FromFunction (string name, NSExpression[] parameters);
 
+		[Static, Export ("expressionWithFormat:")]
+		NSExpression FromFormat (string expressionFormat);
+		
+		[Static, Internal, Export ("expressionWithFormat:arguments:", IsVariadic = true)]
+		NSExpression FromFormat (string expressionFormat, IntPtr argList);
```

Comment:
This is defined as a `va_list` in ObjC, which I believe is architecture-specific, so it can't really be bound in C#.

I think the best would be call the `expressionWithFormat:argumentArray:` selector from the code-behind method instead of this one (and just remove this binding completely).

Binding variadic methods should be avoided whenever possible, because it turns out quite ugly (see https://github.com/xamarin/xamarin-macios/blob/master/src/UIKit/UIAppearance.cs#L112 if you wish).